misc/metrics/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # 0.10.0 [unreleased]
 
+- Added `protocol_stack` metrics.
```

Comment:
All entries apart from dependency bumps should also mention the PR they come from. See some of the other examples in this file :)

Plus, Max just pushed out 0.49, so this will need an update to the master branch and an according version bump (patch-level) please!